app/controllers/coffee_controller.rb

```diff
@@ -1,11 +1,14 @@
 class CoffeeController < ApplicationController
   
   def index
-    @most_recent_brew = Brew.last
+    if current_user 
+      @most_recent_brew = Brew.where(team_id: current_user.team_id).last
```

Comment:
actually what you could do is find_current_team.brews.last but again not a huge deal